docs/UserGuide.md

```diff
@@ -138,7 +134,8 @@ The command `list_session` will show the following:
 * `find_session John` returns all John in TutorBuddy
 * `find_session Jo` returns all John in TutorBuddy
 * `find_session John Lee` returns all John Lee in TutorBuddy
-* `find_session Zach` returns nothing
```

Comment:
As per the above, this seems to follow the find_session s/STUDENT_NAME instead ah, should we comment this out first?

Reply:
The chunk (from line 114 to 138) is commented out using HTML <!-- --> tags! So should be irrelevant for now.